src/rlang/globals.c

```diff
@@ -28,6 +28,7 @@ void r_init_library_globals() {
   r_globals.na_lgl = NA_LOGICAL;
   r_globals.na_int = NA_INTEGER;
   r_globals.na_dbl = NA_REAL;
+  r_globals.na_cpl = (r_complex_t) {NA_REAL, NA_REAL};
```

Comment:
I think so! Just a style nitpick:

```suggestion
  r_globals.na_cpl = (r_complex_t) { NA_REAL, NA_REAL };
```